eng/docker-compose/.env.elasticsearch.example

```diff
@@ -0,0 +1,65 @@
+#
+# Variables for docker-compose.yml
+#
+
+POSTGRES_PASSWORD=P@ssW0rd
+POSTGRES_DB_NAME=edfi_datamanagementservice
+
+CONNECT_SOURCE_PORT=8083
+CONNECT_SINK_PORT=8084
+
+ELASTICSEARCH_HTTP_PORT=9200
+ELASTICSEARCH_ANALYZER_PORT=9300
+ELASTICSEARCH_DASHBOARD_PORT=5601
```

Comment:
Recommendation: put the new settings in to the existing example file instead of creating a new one. Add a comment explaining that you only need either the elastic search or the open search settings.